src/components/devsupport/typings.ts

```diff
@@ -1,12 +1,14 @@
 import { ReactElement } from 'react';
 
 export type ChildrenProp<Element extends ReactElement = ReactElement> = Element | Element[];
+// eslint-disable-next-line @typescript-eslint/ban-types
 export type ChildrenWithProps<Props = {}> = ChildrenProp<ReactElement<Props>>;
```

Comment:
We can get rid of default type value (as we did [here](src/components/devsupport/components/falsyFC/falsyFC.component.tsx) for example)